index.html

```diff
@@ -16,9 +16,11 @@
   </div>
   <hr>
   <div id="notice">
-    <span style="font-size:1.1em; font-weight: bold">Get ready for the November hardfork upgrade!</span>
+    <span style="font-size:1.1em; font-weight: bold">The November protocol upgrade has completed successfully!</span>
     <br>
-    Install the latest version (0.18.3):
+    The release below contains a post-fork checkpoint to improve chain stability and prevent pre-fork reorgs. We highly recommend upgrading to prevent loss of user funds and disruption to your business.
```

Comment:
I'd maybe tone down the wording, it sounds a little scary and negative (loss of funds)

Maybe something like:

"We highly recommend upgrading to ensure safety of funds and stable operation of your business.

Reply:
Good idea

Reply:
I'd also remove "highly"